packages/react-dom/src/__tests__/ReactDOMComponent-test.js

```diff
@@ -1151,19 +1154,37 @@ describe('ReactDOMComponent', () => {
         <input type="text" defaultValue="foo" />,
         container,
       );
+      var getTrackedValue = getValueTracker(inst);
```

Comment:
I'm not sure if you do have to create an instance for the trackers. See, the solution I did for ReactDOMInput-test actually only had the calls for the input values i wanted to test. 

Anyway, I can't see any trouble really about this solution.

Reply:
@SadPandaBear Thank you for reviewing. I have confirmed that changing value using tracker instance still pass the test. The reason I create an instance for the tracker is because  `getValueTracker` is being used for both `input` and `textarea` element. Not sure about using `HTMLInputElement` for `textarea` is the correct way to do it.

Reply:
>Not sure about using HTMLInputElement for textarea is the correct way to do it.

Can you declare two separate helpers (for input and textarea) instead?

Reply:
@gaearon got it, I will define the helper in each test case instead so that `input` will use `HTMLInputElement`, and `textarea` will use `HTMLTextAreaElement`